### ISSUE TITLE
False positive for `Layout/ClosingParenthesisIndentation` cop in 0.70.0 ?

### ISSUE THREAD

**Issue by mjankowski** [sds/haml-lint#629]
When I update to 0.70.0, in a view like this (I'm not original author, I'm just attempting haml-lint update...):

```haml
-# locals: (status:, is_quote: false)
- if status.spoiler_text.present?
  %details<
    %summary{
      data: {
        show: t('statuses.content_warnings.show'),
        hide: t('statuses.content_warnings.hide'),
      }
    }><
      %strong>
        = prerender_custom_emojis(h(status.spoiler_text), status.emojis)
    .status__content><
      = prerender_custom_emojis(status_content_format(status), status.emojis)
    = render partial: 'admin/shared/status_attachments', locals: { status: status.proper, is_quote: is_quote }
- else
  .status__content><
    = prerender_custom_emojis(status_content_format(status), status.emojis)
  = render partial: 'admin/shared/status_attachments', locals: { status: status.proper, is_quote: is_quote }
```

I get an error with the `Layout/ClosingParenthesisIndentation` cop failing. It's complaining about line 6, which is the `show: ...` line within the `data` attributes (though with the first comment line and general haml->ruby conversion I'm never totally confident in line numbers).

I'm able to fix the error by arranging the hash alignment differently, putting it all on one line, etc.

Curious if this is an intentional failure or a correction of a previous false negative ... or if this was unintentional side effect of some other change?

Related: https://github.com/mastodon/mastodon/pull/37902

**Comment by maddieholtzer** [comment]
Experiencing this same issue.

Unsure if it is related but am experiencing a weird issue with this where the base `haml-lint` command flags the `Layout/ClosingParenthesisIndentation` lints but `haml-lint -A` does not:

```
$ haml-lint
916 files inspected, 35 lints detected

$ haml-lint -A
916 files inspected, 0 lints detected
```

This does not appear to be happening for other lints:

```
$ haml-lint some_file.html.haml
some_file.html.haml:203 [W] RuboCop: Layout/ExtraSpacing: Unnecessary spacing detected.
some_file.html.haml:209 [W] RuboCop: Layout/ClosingParenthesisIndentation: Indent `)` to column 12 (not 0)
1 file inspected, 2 lints detected

$ haml-lint some_file.html.haml -A
some_file.html.haml:203 [W] [Corrected] RuboCop: Layout/ExtraSpacing: Unnecessary spacing detected.
1 file inspected, 1 lint detected, 1 lint corrected
```
